modules/layers/src/screen-grid-layer/screen-grid-layer.js

```diff
@@ -322,7 +344,10 @@ export default class ScreenGridLayer extends Layer {
       gridTransformMatrix
     });
 
-    const {maxWeight = 0} = results;
```

Comment:
Should the call to gpuGridAggregator.run be changed? It appears that the countsBuffer and maxCountBuffer passed into it/the buffer should is passed via the weights object.

Reply:
Yes, please check line 88 and 412.